tests/mol-timing-synchronization_fxl/OPS/content_001.xhtml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+        <meta charset="utf-8" />
+		<title>Media overlay and fixed layout</title>
+        <meta name="viewport" content="width=900, height=600" />
+	</head>
```

Comment:
Arrgh... this is because these tests come somewhere else and they clash with the way my editor seems to work...

I also removed the utf meta (and the xml declaration that we do not use elsewhere).